backend/src/BuiltinExecution/Libs/Int128.fs

```diff
@@ -0,0 +1,371 @@
+module BuiltinExecution.Libs.Int128
+
+open FSharp.Control.Tasks
+open System.Threading.Tasks
+
+open System.Numerics
+
+open Prelude
+open LibExecution.RuntimeTypes
+open LibExecution.Builtin.Shortcuts
+
+module VT = ValueType
+module Dval = LibExecution.Dval
+
+let types : List<BuiltInType> = []
+
+let modules = [ "Int128" ]
+let fn = fn modules
+let constant = constant modules
+
+let constants : List<BuiltInConstant> =
+  [ { name = constant "zero" 0
```

Comment:
The reason for adding this constant is its use within the `Int128.absoluteValue` function, specifically in the line `if (Builtin.Int128.lessThan a Builtin.Int128.zero) then ...`

**Using 0 instead of Builtin.Int128.zero :** I get an error stating that Int128.lessThan's 2nd argument should be an Int128. However, an Int (0) was passed instead.

**Using 0Q instead of Builtin.Int128.zero :** The loading packages to canvas step fails with error saving function to canvas
details:
  ```
  Type: function; Json: {"body":{"EIf":[5413835516771744658,{"EApply":[5967867546811000111,{"EFnName":[8251259138368260562,{"Ok":[{"BuiltIn":[{"modules":["Int128"],"name":{"FnName":["lessThan"]},"version":0}]}]}]},[],[{"EVariable":[4596083514880703310,"a"]},{"EInt128":[6792095667907226909,0]}]]},{"EApply":[3771221013789687638,{"EFnName":[2615256928638251462,{"Ok":[{"BuiltIn":[{"modules":["Int128"],"name":{"FnName":["negate"]},"version":0}]}]}]},[],[{"EVariable":[727526345080291056,"a"]}]]},{"Some":[{"EVariable":[2686468324135877090,"a"]}]}]},"deprecated":{"NotDeprecated":[]},"description":"","id":"2934c4f0-c135-4d77-a87d-3443a912e516","name":{"modules":["Stdlib","Int128"],"name":{"FnName":["absoluteValue"]},"owner":"Darklang","version":0},"parameters":[{"description":"","name":"a","typ":{"TInt128":[]}}],"returnType":{"TInt128":[]},"tlid":6253735342452977547,"typeParams":[]}
Status: 500
```

Reply:
Well we should be able to use 0Q, so it looks like there's something broken there.

Reply:
But yeah, we should remove this.

Reply:
Please correct me if I'm wrong, but I think the issue is that we cannot currently save Int128 in the database. As a result, we fail when attempting to make a post request to save the absoluteValue function to the PackagesDB?